web/src/pages/suppRequests/ApplicationFinalized.tsx

```diff
@@ -0,0 +1,75 @@
+// (c) Tecnologico de Monterrey 2022, rights reserved.
+
+import RequestCard from "../../components/RequestCard/RequestCard";
+import back from "../../assets/background/login.png";
+import { useLazyLoadQuery } from "react-relay";
+import graphql from "babel-plugin-relay/macro";
+
+import {
+  ApplicationFinalizedQuery,
+  ApplicationFinalizedQuery$data,
+} from "./__generated__/ApplicationFinalizedQuery.graphql";
+
+const ApplicationFinalized = (): JSX.Element => {
+  const data: ApplicationFinalizedQuery$data = useLazyLoadQuery<ApplicationFinalizedQuery>(
+    graphql`
+      query ApplicationFinalizedQuery($application_status_id: ID!) {
+        applicationByStatusID(application_status_id: $application_status_id) {
+          title
+          id
+          user {
+            id
+            name
+            first_lastname
+            second_lastname
+          }
+          citation {
+            id
+            title
+          }
+        }
+      }
+    `,
+    { application_status_id: "" },
+  );
+
```

Comment:
If query arguments can be sent null modify schema and handle null values inside query resolver.